_posts/0000-01-02-longzuchihuo.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```

Comment:
The next field helps you provide a description of the changes you made. 
Feel free to add a description of what you’ve accomplished so far. 
As a reminder, you have: created a branch, created a file and made a commit, and opened a pull request
The next field helps you provide a description of the changes you made. 
Feel free to add a description of what you’ve accomplished so far. 
As a reminder, you have: created a branch, created a file and made a commit, and opened a pull request